projects/core/src/lib/layers/legend/legend-raster/raster-legend.component.spec.ts

```diff
@@ -46,4 +46,48 @@ describe('RasterLegend', () => {
         ];
         expect(calculateNumberPipeParameters(breakpoints6)).toBe('1.0-0');
     });
+
+    it('unifyDecimals', () => {
```

Comment:
There is an error running this test in the CI.
Please first let the CI run through before assigning the PR for review.